content/faq/faq.mdx

```diff
@@ -100,21 +100,56 @@ const RatingControl = ({ data, handleChange, path }) => (
   />
 );
 
+const renderers = [
+    ...materialRenderers,
+    //register custom renderer
+    { tester: rankWith(3,scopeEndsWith('rating')), renderer: RatingControl }
+  ]
+
+...
+
 <JsonForms
   schema={schema}
   uischema={uischema}
   data={stateData}
-  renderers={[
-    ...materialRenderers,
-    //register custom renderer
-    { tester: rankWith(3,scopeEndsWith('rating')), renderer: RatingControl }
-  ]}
+  renderers={renderers}
 />
 ```
 
 For more information about custom renderers, have a look [here](/docs/tutorial/custom-renderers).
 
-## How can I use default values within JSON Forms
+## How can I minimize re-rendering in React?
+
+JSON Forms uses React.memo, to avoid any unnecessary re-rendering stable properties should be handed over.
```

Comment:
```suggestion
JSON Forms uses React.memo to avoid any unnecessary re-rendering.
Therefore props should in general be stable, for example by memoizing them.
```